src/views/Admin/Modules/Records/List.vue

```diff
@@ -114,5 +121,28 @@ export default {
       rowCreateUrl: 'admin.modules.record.create',
     }
   },
+
+  methods: {
+    ...mapActions({
+      updateModule: 'module/update',
+    }),
+
+    handleSave (fields = []) {
+      fields = JSON.parse(JSON.stringify(fields))
+      if (!this.module.meta.ui) {
+        this.module.meta.ui = {
+          admin: {
+            fields,
+          },
+        }
+      } else {
+        this.module.meta.ui.admin = { fields }
+      }
+
+      this.updateModule(this.module).then(() => {
+        this.toastSuccess(this.$t('notification.saved'))
```

Comment:
Lets change this notification to something more fitting, since we are just saving columns on all records here and not the module (from user perspective)